options/options.go

```diff
@@ -302,8 +302,40 @@ func (terragruntOptions *TerragruntOptions) Clone(terragruntConfigPath string) *
 	}
 }
 
+// Check if argument is planfile TODO check file format
+func checkIfPlanfile(arg string) bool {
+	stat, err := os.Stat(arg)
+
+	if err != nil {
+		return false
+	}
+
+	return !stat.IsDir()
+}
```

Comment:
You can use [util.IsFile](https://github.com/gruntwork-io/terragrunt/blob/master/util/file.go#L130) for checking if file.

Reply:
Hi @yorinasub17 ,
fixed